packages/docusaurus-preset-classic/src/index.ts

```diff
@@ -48,6 +48,16 @@ export default function preset(
   if (algolia) {
     themes.push(require.resolve('@docusaurus/theme-search-algolia'));
   }
+  if ('gtag' in themeConfig) {
```

Comment:
why is this addition needed?

Aren't plugins already throwing when running their own themeConfig validation? If it's not the case we have another deeper problem

Reply:
> Edit. I see the reason now. If there's not a googleAnalytics in preset options, the plugin isn't even loaded and there's no theme config validation either

Oh I see, nasty 😅 Thanks for the fix